ct.yaml

```diff
@@ -0,0 +1,5 @@
+chart-dirs:
```

Comment:
My only thought here is that chart-testing is fairly new for us, so the `ct.yaml` config file feels a bit obscure. Maybe we can add a comment at the top saying

```yaml
# chart-testing configuration for chart-testing-action (GitHub Actions)
# Docs: https://github.com/helm/chart-testing#configuration
```